### ISSUE TITLE
Request for list of randomly selected scenes

### ISSUE THREAD

**Issue by Fusroda-h** [xiliu8006/3dgs_dataset_generator#2]
Thank you for your fascinating work!

I am currently working on our project,
and need **the list of scenes used in the randomly selected 120 training set and 30 test set.** 
Could you please provide this list?

I appreciate your help.
Thank you!

**Comment by fgz-ai** [comment]
Hi,
I have the same request..
Providing the list of selected training and test scenes enables direct comparison with your method and allows us to properly cite your paper.

Thanks,
